### ISSUE TITLE
Issue with level-type in Dev version

### ISSUE THREAD

**Issue by CTO7** [NillerMedDild/Enigmatica6#2424]
**Modpack version**
Github 0.5.0

**Describe the bug**
When setting up the server files in server.properties the line
`level-type=terraforged `
has a space after it which causes a crash on start

**Logs**
Input pastebin link to crash-report or `latest.log`, if applicable.
https://pastebin.com/SF1uQ8YP

**Screenshots**

**Comment by NillerMedDild** [comment]
I can't reproduce this issue, how did your server.properties generate, which file did you  use to start the server? :)

**Comment by anubiann00b** [comment]
I'm getting the same issue--here's the steps I took:

- cloned the repo fresh
- ran ./InstanceSetupSync.bat
- ran ./update-server.ps1
- ran ./start-automated-server.bat

![image](https://user-images.githubusercontent.com/5544386/122270551-48185280-ce93-11eb-9e11-da7716a35e0b.png)

Removing the trailing space fixes the problem for me. Maybe related, but I noticed when setting things up this way that eula.txt also had a trailing space that prevented it from starting? The startup script didn't warn for it, but the minecraft server itself exiting with an error message about it.
